nginx_config_reloader/__init__.py

```diff
@@ -400,7 +374,11 @@ def process_IN_DELETE(self, event):
     )
 
     if nats_server:
-        start_message_subscribe_loop(nginx_config_changed_handler, nats_server)
+        nats_thread = threading.Thread(
```

Comment:
Does this work with async? Shouldn't you use `asyncio.get_running_loop()` instead?